### PR TITLE
[WIP] 426 breadcrumb nav

### LINKED ISSUE #422: "Create New Buttons" at the top of the list
As a reporter (Björn)
i want to have the buttons for "Create Event" and "Create Trigger"  at the top oft the list (or as a button (not link) in the grey menu box)
in order to faster create these elements.

### LINKED ISSUE #426: Breadcrumb Navigation
As a reporter
I prefer to have a bredcrumb navigation (as proposed in #404)
in order to stay in routing for my report.

### PR DESCRIPTION
**Pull request by tillprochaska**
close #426 
close #422

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -20,4 +21,124 @@ def set_report
...
+        name: 'Report Settings',
+        url: report_path(report),
+        active: params[:controller] == 'reports' && ['show', 'edit'].include?(params[:action])
+      }
+
+      action = 'index'
+
+      if params[:controller] == 'reports' && ['present', 'preview'].include?(params[:action])
+        action = params[:action]
+      elsif params[:controller] == 'reports' && params[:action] == 'show'
```
Comment (roschaefer) [inline app/controllers/application_controller.rb]: nest if cases, in general avoid if cases

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -20,4 +21,124 @@ def set_report
...
+      }
+    end
+
+    actions['events'] = {
+      name: 'New Event',
+      url: url_for(controller: 'events', action: 'new')
+    }
+
+    primary_action = params[:controller]
+    if !actions.keys.include? params[:controller]
```
Comment (roschaefer) [inline app/controllers/application_controller.rb]: use `unless`

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -21,72 +21,14 @@
...
-
-          .card.bg-light
-            .card-header Switch report
-            .card-body
-              %ul.nav.nav-list.flex-column.report-menu
-                - Report.find_each do |report|
-                  %li.nav-item= link_to report.name, present_report_path(report)
+    .container
       %footer.py5
-        = "story.board. The CMS for Sensorstories. Created 2015 - #{Time.zone.today.year} by Robert Schäfer and Jakob Vicari. No Reporters Were Harmed."
```
Comment (roschaefer) [inline app/views/layouts/application.html.haml]: Use `Time.zone`

**Comment by drjakob** [comment]
Good job!

**Pull request merged**